### PR TITLE
"official" mobile port

### PR DESCRIPTION
**Pull request by ghost**
squshed commit

### CODE REVIEW

**Comment by Vortex2Oblivion** [comment]
You do realize you can edit a pr after you made it right?
You didn't need to make a new one

**Comment by ghost** [comment]
git fucked local repo sadly

File: Project.xml [hunk 1]
```diff
@@ -128,7 +132,10 @@
 	<haxedef name="HXCPP_STACK_LINE" if="release" />
 	<haxedef name="HXCPP_STACK_TRACE" if="release" />
 
-	<haxedef name="HXCPP_GC_BIG_BLOCKS" />
+	<!-- Do whatever you want I'm tired uninstall and install everytime -->
+	<certificate path="key.keystore" password="leatherengine" alias="leatherport" alias-password="leatherengine" if="android" unless="debug" />
+
+	<!--<haxedef name="HXCPP_GC_BIG_BLOCKS" />-->
```
Comment (Vortex2Oblivion) [inline Project.xml:138]: Why are you commenting out this define

**Review by Vortex2Oblivion — CHANGES_REQUESTED** [review]

File: Project.xml (hunk 1 above)
Comment (ghost) [reply Project.xml:138]: it causes memory leaks due big block creation

Comment (Vortex2Oblivion) [reply Project.xml:138]: K

**Comment by Vortex2Oblivion** [comment]
Oops I marked it as ready for review on accident lol

**Comment by Vortex2Oblivion** [comment]
Can you change it back to a draft by any chance?

**Comment by Vortex2Oblivion** [comment]
@what-is-a-git Can you look at this when you get the chance?

**Comment by ghost** [comment]
@Vortex2Oblivion @what-is-a-git anything?

**Comment by cherrythecool** [comment]
I'm not personally going to look into this at the moment but if Vortex wants to he can go ahead.

**Comment by Vortex2Oblivion** [comment]
I dont think im going to merge this

**Comment by ghost** [comment]
that's okay

**Pull request closed without merging**